site/src/components/FooterBanner.tsx

```diff
@@ -9,7 +9,6 @@ import {
 } from "@mui/material";
 import Image from "next/image";
 import { Link } from "./Link";
-import backgroundRainbow from "../../public/assets/background-corner-rainbow.png";
```

Comment:
This did not work, probably because of interference with `next-images`. I’ve switched to `src="/assets/..."` as this is what we do in a few other places. We can probably remove `next-images` or change its config as a [follow-up task](https://app.asana.com/0/0/1201815866759884/f) (internal link).